src/ControlSystem/Tags/QueueTags.hpp

```diff
@@ -88,6 +89,34 @@ struct InverseSpatialMetricOnExcisionSurface {
   using type = tnsr::II<DataVector, 3, Frame>;
 };
 
+/// \ingroup ControlSystemGroup
+/// Holds the spatial christoffel symbol on the `ExcisionSurface`
+template <typename Frame>
+struct SpatialChristoffelSecondKind {
+  using type = tnsr::Ijj<DataVector, 3, Frame>;
+};
+
+/// \ingroup ControlSystemGroup
+/// Holds the spatial derivative of the lapse on the `ExcisionSurface`
+template <typename Frame>
+struct DerivLapse {
+  using type = tnsr::i<DataVector, 3, Frame>;
+};
+
+/// \ingroup ControlSystemGroup
+/// Holds the spatial derivative of the `Frame` shift on the `ExcisionSurface`
+template <typename Frame>
+struct DerivShift {
+  using type = tnsr::iJ<DataVector, 3, Frame>;
+};
+
+/// \ingroup ControlSystemGroup
+/// Holds the inverse jacobian between frames on the `ExcisionSurface`
+template <typename SrcFrame, typename DestFrame>
+struct InverseJacobian {
+  using type = ::InverseJacobian<DataVector, 3, SrcFrame, DestFrame>;
+};
+
```

Comment:
Add tests for these tags?

Reply:
These aren't DataBox tags. Though I guess I could just use the existing GR tags for this

Reply:
Oh, right. I would say just leave as-is? Could also just inherit off `db::SimpleTag`? I don't have a preferences

Reply:
I think I'll leave as is so it's the same as the other Queue tags in this file